### ISSUE TITLE
in pattern table column is named organisedId and not organiserId

### ISSUE THREAD

**Issue by bendixsonnenberg** [cs-select-team/CS-Select#223]
![image](https://user-images.githubusercontent.com/25992157/54043214-56fbff00-41cc-11e9-88d7-87b4ceab60b2.png)
and this leads to the server throwing a exception 
```
com.csselect.database.DatabaseException: java.sql.SQLSyntaxErrorException: Unknown column 'organiserId' in 'where clause'
	com.csselect.database.mysql.MysqlDatabaseAdapter.executeMysqlQuery(MysqlDatabaseAdapter.java:354)
	com.csselect.database.mysql.MysqlDatabaseAdapter.executeMysqlQuery(MysqlDatabaseAdapter.java:328)
	com.csselect.database.mysql.MysqlOrganiserAdapter.getPatterns(MysqlOrganiserAdapter.java:55)
	com.csselect.user.Organiser.getPatterns(Organiser.java:54)
	com.csselect.API.APIFacadeOrganiser.getPatterns(APIFacadeOrganiser.java:118)
	com.csselect.API.httpAPI.GameManagement.getPatterns(GameManagement.java:130)
	com.csselect.API.httpAPI.GameManagement.get(GameManagement.java:28)
	com.csselect.API.httpAPI.Servlet.doGet(Servlet.java:93)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:634)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
```

**Comment by bendixsonnenberg** [comment]
also prevents gamecreation

**Comment by SeineEloquenz** [comment]
There was a typo in the table names which was fixed in 3c905363b76fad924a81aa96aefa917642b98c0a
This might result from switching between branches where this fix is not yet included

**Comment by bendixsonnenberg** [comment]
in that case I'll close this
